src/index.js

```diff
@@ -203,13 +221,19 @@ export function validate(config, metaConfig) {
 }
 
 function assertValid(value, validateKey, validator) {
-    if (!assert(value, validator)) {
-        throwError(validateKey);
+    const result = assert(value, validator);
```

Comment:
I think we should change the name of "assert" now, as the behaviour has changed.